app/views/fields/belongs_to/_show.html.erb

```diff
@@ -16,5 +16,5 @@ By default, the relationship is rendered as a link to the associated object.
 %>
 
 <% if field.data %>
-  <%= link_to field.data, [Administrate::NAMESPACE, field.data] %>
+  <%= link_to field.associated_resource, [Administrate::NAMESPACE, field.data] %>
```

Comment:
If I were to describe this in English, it'd be "the human-readable representation of the resource that this field points to".

So `display_associated_resource` is *kind of* saying the same thing. But I can't think of a better name, honestly.

It would be nice if `associated_resource` were a decorator around a record instead of a record itself. Actually, every resource that needs to be `display`ed could be decorated, and then you could have your `to_s` method again, only inside the decorator class. I guess it wouldn't be easy for a user to override that, though.